backend/cli.py

```diff
@@ -111,19 +131,22 @@ def import_gerrydb_view(layer: str, gpkg: str, replace: bool, rm: bool):
     _session = get_session()
     session = next(_session)
 
-    upsert_query = text("""
+    upsert_query = text(
+        """
         INSERT INTO gerrydbtable (uuid, name, updated_at)
-        VALUES (gen_random_uuid(), :name, now())
+        VALUES (gen_random_uuid(), :name, :extent, now())
         ON CONFLICT (name)
         DO UPDATE SET
             updated_at = now()
```

Comment:
H: You're not actually setting the extent on conflict. Not sure if that's intentional or not but if the extent was null in an existing table it would remain null. Also you have a mismatch in the number of inserted values and columns specified so this would fail anyway.